pages/clubs-guides/clubs-guides.jsx

```diff
@@ -0,0 +1,99 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+
+var GuideList = React.createClass({
+  getInitialState: function(){
+    return {
+      guides : []
+    }
+  },
+  setData : function(data){
+    this.setState({guides : data });
+  },
+  fetchJSON : function(path,callback){
+    var httpRequest = new XMLHttpRequest();
+    httpRequest.onreadystatechange = function() {
+      if(httpRequest.readyState === 4) {
+        if(httpRequest.status === 200) {
+          var data = JSON.parse(httpRequest.responseText);
+          if (callback) callback(data);
+        }
+      }
+    };
+    httpRequest.open('GET', path);
+    httpRequest.send();
+  },
+  componentDidMount: function() {
+    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+  },
+  render: function(){
+    var categories = this.state.guides.map(function(e){
+
+      var guideLinks = e.guides.map(function(f){
+        var translationLinks = [];
```

Comment:
nit: you don't need to set it to `[]`
and then you just do `if(translationLinks)` [here](https://github.com/mozilla/learning.mozilla.org/pull/1976/files#diff-6ce391f406c6e2e4b79cec1edd0c1940R46)